lib/navigation/app_router.dart

```diff
@@ -11,17 +12,23 @@ enum RoutePath {
   root('/'),
   home('/home'),
   details('details'),
-  signIn('/sign_in');
+  signIn('/sign_in'),
+  questions('questions/:$surveyIdKey');
```

Comment:
@hoangnguyen92dn it's better to group the parent route (start with `/`) together, and the sub-route should be below (non `/` at the start).